src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1661,6 +1662,7 @@ DeviceConsent PolicyHandler::GetUserConsentForDevice(
 }
 
 Json::Value PolicyHandler::GetPolicyTableData() const {
+  POLICY_LIB_CHECK(Json::Value());
```

Comment:
@YarikMamykin If you decided to use MACRO - make it more clear. 
I mean make POLICY_LIB_CHECK() for void functions and POLICY_LIB_CHECK_OR_RETURN() for non-void ones. 
Ii is not obvious in the code why do you pass parameters of different types to the macro.

Reply:
@AByzhynar reworked in https://github.com/smartdevicelink/sdl_core/pull/3292/commits/f037262a5638e889e9533a5ec17ac10e286176fa